### ISSUE TITLE
Stopping mockserver fails to callback is not a function

### ISSUE THREAD

**Issue by Humakt83** [jamesdbloom/mockserver-grunt#12]
Hey.

I'm utilizing mockserver for protractor tests to mock a server. Runs otherwise fine but stopping the mock server fails due to following error:
```

Using port '8500' to stop MockServer and MockServer Proxy
TypeError: callback is not a function
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickCallback (node.js:349:13)
[09:39:24] E/launcher - Process exited with error code 1
```

Using node v4.6.1 on Linux machines

**Comment by jamesdbloom** [comment]
It may be due to the protractor test finishing before the promise has completed, this is an issue I've seen before with the mockserver-client-node due to using Q instead of the built in promise in protractor.  I'll make a commit that should in theory fix that, but I have no easy way to test it so please retest and let me know if that worked.

**Comment by jamesdbloom** [comment]
This change is available in the latest release (1.0.37) so please upgrade and re-test.

**Comment by jamesdbloom** [comment]
I'm assuming this is now fixed
